docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+
+We'll be working primarily in `main.tsx` and `styles.css` located in the `src/` folder, which (after running `npm start` as described in the previous tutorial) get recompiled to the `dist/` folder after every save. This is an important fact to keep in mind; for one thing, you'll want to place whatever 3D models and various media files you'll be using in the `dist` folder since that'll be the root of your local references - where `main.tsx` will be solving paths from.
+
+Now, let's shave down the default scene graph created in `main.tsx` to a minimum, so that it only contains the bare bones:
+
+```tsx
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import bind from 'bind-decorator';
+
+import { AvGadget, AvTransform, AvPanel, AvGrabbable, 
+HighlightType, GrabResponse, AvSphereHandle, 
+AvTransformControl, AvPanelAnchor, AvModel } from '@aardvarkxr/aardvark-react';
+
+import { EndpointAddr, AvGrabEvent, endpointAddrToString } from '@aardvarkxr/aardvark-shared';
+
+interface WatchState {}
+
+class WatchGadget extends React.Component<{}, WatchState>
+{
+	constructor(props: any) {
+	super(props);
+	}
+
+	public render() {
+		let scale = 0.2;
+
+return (
+	<div>
+		<AvGrabbable dropOnHooks={true}>
+			<AvSphereHandle radius={0.1} />
+			<AvTransform uniformScale={0.2}>
+				<AvPanel interactive={true} >
+				</AvPanel>
+			</AvTransform>
+		</AvGrabbable>
+	</div>
+		)
+	}
+}
+ReactDOM.render(<WatchGadget />, document.getElementById("root"));
+```
+
+You'll notice we've kept a few things that will be immediately useful: 
+*  `<AvSphereHandle />`, because we'll want to interact with our gadget - no sense in losing the handler. 
+*  `<AvTransform>` to easily manipulate our panel, as well as the contained panel.
+
+Aside from that, and new empty WatchState object and a few naming tweaks, this is a barebones version of the default gadget. Let's get to work!
+
+First, right before the final `</div>` tag in the `render` method, let's add a simple sanity check to make sure our dev environment is fully functional and the gadget is being updated:
+
+```tsx
+<div className="Watch">
+	Hello Aardvark!
+</div>
+```
+Upon saving and instantiating a new gadget, you should be able to see immediate results, but they'll default to the size of fairly small chicken scratch. In `styles.css`, let's fix that!
+
+```css
+.Watch
+{
+	font-size: 4rem;
+}
+```
+![hello-aardvark](./media/hello-aardvark.jpg)
+
+Now we're cooking with fire. Let's get it looking like a proper digital watch. We'll tweak the CSS of the `.Watch` class so that the numbers utilize a proper digital font (I picked the freely available [Digital 7](https://www.1001fonts.com/digital-7-font.html), and position it somewhere that'll make sense once we add the digital watch model.
+
+A nice trick here is making the panel background transparent, because we don't want a "page" as much as "digits displayed on a watch's screen". Here's `styles.css`:
+
+```css
+@font-face {
```

Comment:
Opening brace on the new line please

Reply:
ugh, sorry about that. need to find a good CSS linter.